datasets/colours.py

```diff
@@ -71,10 +82,21 @@ def generate_examples(self, num_examples):
         return examples
 
     def evaluate_correct(
-        self, questions: List[str], responses: List[str], expected_answers: List[str]
+            self, questions: List[str], responses: List[str], expected_answers: List[str]
     ) -> Tuple[int, int, List[str]]:
-        color = expected_answers[0].lower()
-        if color in responses[-1].lower():
-            return 1, 1, [f'"{color}" is in the response.']
-        return 0, 1, [f'"{color}" is NOT in the response.']
+        score = 0
+        max_score = len(expected_answers)
```

Comment:
`max_score` will be deprecated at some point, but for now we should set it always to `1`.

Reply:
Will revise accordingly